koji_wrapper/wrapper.py

```diff
@@ -4,12 +4,19 @@
 Wraps a connection to koji, managing the session, and providing convenience
 methods for interacting with the koji api.
 """
+import os
 
 import koji
+
 from koji_wrapper.base import KojiWrapperBase
 
+
 class KojiWrapper(KojiWrapperBase):
 
+    def __init__(self, **kwargs):
+        self._pathinfo = None
+        super(KojiWrapper, self).__init__(**kwargs)
```

Comment:
the newer python style for this is:
``
super().__init__(**kwargs)
``

I have already used this in my patch for KojiTag #18, so I would like to keep this consistent (plus, I think it is a nicer syntax anyway)

Reply:
personal style, which could have been rolled into a follow up patch to keep things consistent but fixed.